app/views/ideas/show.html.erb

```diff
@@ -81,3 +81,50 @@
 <% end %>
 <div class="govuk-!-padding-bottom-5"></div>
 <%= link_to 'Back', ideas_path %>
+
+<script>
+  function vote(event) {
```

Comment:
I find the indentation inside this method hard to follow.  Do you have an auto-formatter configured which can fix this for you?  If not give me a shout and I can show you.

Reply:
The vote function is quite long.  It would be good to investigate refactoring some of the functionality into smaller methods.